test/graphql/Applicant/queries/getApplicants.test.ts

```diff
@@ -108,7 +118,11 @@ describe("getApplicants", () => {
         apolloClient
       } = await testClientFactory.applicant({
         careers: applicantCareersData,
-        capabilities: ["Go"]
+        capabilities: ["Go"],
```

Comment:
Quizas a este test lo renombraria a `allows to fetch all the applicants` asi queda claro que un applicant aprobado o un admin pueden hacer esto